app/views/posts/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>This is all posts of users</h1>
\ No newline at end of file
+<h1 class="post_heading">This is all posts of users</h1>
+
+<div id="postTemplate">
+   <%= render partial: "shared/single_user", locals: { user: @user } %>
+<% @posts.each do |post| %>
+     <%= render partial: "shared/single_post", locals: { post: post } %>
+     <%= render partial: "shared/collection_comments", locals: { post: post, user: @user } %>
+<% end %>
+<%= button_to "Paginations"%>
```

Comment:
* Kindly use `post.recent_comments`  method to render 5 most recent comments, you have this methods created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md). to render the user's 3 most recent posts. [as mentioned in the project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.)